assets/src/util/sentry.tsx

```diff
@@ -19,6 +19,65 @@ const captureException = (ex: unknown, options?: RavenOptions) => {
   Raven.captureException(ex, options);
 };
 
+const offline_events_key = "sentry_offline_events";
```

Comment:
Let's name this `OFFLINE_EVENTS_KEY` so in the places where it's used, it's obvious it's referring to a constant value rather than a local variable.